app/uk/gov/hmrc/platopsgithubproxy/service/GitHubProxyService.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.platopsgithubproxy.service
+
+import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse, UpstreamErrorResponse}
+import uk.gov.hmrc.platopsgithubproxy.connector.GitHubConnector
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.Future
+
+@Singleton()
+class GitHubProxyService @Inject()(
```

Comment:
Do we need this extra layer of indirection? Looks like just pass through

Reply:
Removed